homeassistant/components/pvpc_hourly_pricing/sensor.py

```diff
@@ -142,11 +142,11 @@ def __init__(
         self._attr_unique_id = unique_id
         self._attr_name = name
         self._attr_device_info = DeviceInfo(
-            configuration_url="https://www.ree.es/es/apidatos",
+            configuration_url="https://api.esios.ree.es",
```

Comment:
This isn't required for the library change, right?

Reply:
About the _dance_ around the data-sources of this integration ('esios' to 'apidatos' to 'esios' back again), some context:
- the 'esios_public' (api.esios.ree.es) has been the main one since the first version
- last year, the servers tried to bane HA users (long unrelated story already solved), and the integration changed to implement a _secondary_ data-source (working with the 2 of them). Finally, the secondary source became the default 🙈
- Now I'm removing the secondary data-source, and **implementing the private API** of the same api.esios.ree.es server, with an API Token. The point of it being to **simplify** both the library and the integration and at the same time implementing **features requested by users** for a long time (like sensors for the price of the excess energy produced with PV, etc.)

Reply:
Did this change in the new library release? I don't see where in the diff.